featuretools/primitives/base/primitive_base.py

```diff
@@ -51,3 +57,23 @@ def get_function(self):
 
     def get_filepath(self, filename):
         return os.path.join(config.get("primitive_data_folder"), filename)
+
+    def get_args_string(self):
+        def valid(arg):
+            error = '"{}" must be attribute of {}'
+            assert hasattr(self, arg.name), error.format(arg.name, self.__class__.__name__)
+            is_positional_or_keyword = arg.kind == arg.POSITIONAL_OR_KEYWORD
+            is_same_type = isinstance(getattr(self, arg.name), type(arg.default))
+            default = is_same_type and arg.default == getattr(self, arg.name)
+            return is_positional_or_keyword and not default
+
+        string = OrderedDict()
+        args = signature(self.__class__).parameters.values()
+        for arg in args:
+            if not valid(arg):
+                continue
+            string[arg.name] = str(getattr(self, arg.name))
+        if len(string) == 0:
+            return ''
+        string = ', ' + ', '.join(map('='.join, string.items()))
```

Comment:
can we break this into two lines to make it easier to read

Reply:
Change applied in [this commit.](https://github.com/Featuretools/featuretools/pull/481/commits/38c542d102973819a5a0ce22cbe692dad1c4a340#diff-65ffde62fc03046613de064c794ed250R86)